conf/helmfile.d/0270.polaris-consumer.yaml

```diff
@@ -0,0 +1,92 @@
+releases:
+
+################################################################################
+## Polaris Consumer #############################################################
+################################################################################
+
+#
+# References:
+#   - https://github.com/vanvalenlab/kiosk-console/blob/master/conf/charts/redis-consumer/values.yaml
+#
+- name: polaris-consumer
+  namespace: deepcell
+  labels:
+    chart: redis-consumer
+    component: deepcell
+    namespace: deepcell
+    vendor: vanvalenlab
+    default: true
+  chart: '{{ env "CHARTS_PATH" | default "/conf/charts" }}/redis-consumer'
+  version: 0.6.0
+  values:
+    - replicas: 1
+
+      image:
+        repository: vanvalenlab/kiosk-redis-consumer
+        tag: latest
```

Comment:
Jut making a note so that we don't forget to bump the version to a specific release when the other PRs are done.